spec/lib/api/2.0/skus-spec.js

```diff
@@ -0,0 +1,360 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Skus', function () {
+    var workflowApiService;
+    var _;
+    before('start HTTP server', function () {
+        this.timeout(5000);
+        workflowApiService = {
+            createAndRunGraph: sinon.stub()
+        };
+        helper.setupInjector([
+            helper.require("/lib/services/sku-pack-service"),
+            dihelper.simpleWrapper(function() { arguments[1](); }, 'rimraf')
+        ]);
+        return helper.startServer([
+            dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
+        ]).then(function() {
+            _ = helper.injector.get('_');
```

Comment:
Your code is correct, but actually you can use the `_` directly, you don't have to load it from di, because we have loaded it in spec/helper.js into global context.